jans-cli-tui/cli_tui/plugins/999_jans/main.py

```
@@ -50,20 +68,25 @@ def exit_cli(self) -> None:
         """Removes auth token and exits
         """
 
+        if clear_all:
+            for key in list(config_cli.config['DEFAULT'].keys()):
+                config_cli.config.remove_option('DEFAULT', key)
+        else:
+            config_cli.config['DEFAULT'].pop('access_token_enc', None)
+            config_cli.config['DEFAULT'].pop('user_data', None)
+        config_cli.write_config()
+
         async def coroutine():
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `coroutine`

Add return type annotation: `None`

(ANN202)